### DISCUSSION TITLE
test_node_url更改好像在面板不生效

### DISCUSSION THREAD

**Discussion by TwoBall** [kkkgo/PaoPaoGateWay#24] [问和答]
我改了test_node_url，面板里依旧是YouTube的那个链接。造成我有个被谷歌识别为中国大陆的节点测速一直没有。

**Comment by TwoBall** [comment]
再在面板里改一下又好了。

**Comment by kkkgo** [comment]
`test_node_url`是用于`fast_node`的设置而不是面板的设置。如果面板使用的默认url设置，初次启动面板默认url会被替换成`test_node_url`。

**Reply by kkkgo** [reply]
就是说，面板的设置是存储在浏览器本地的，但如果你刷新缓存或者换个浏览器打开，获取到的设置应该是`test_node_url`一样的设置。

**Reply by kkkgo** [reply]
另外更改也需要`sleeptime`之后生效，默认30秒。

**Comment by kkkgo** [comment]
你修改`test_node_url`配置后，等30秒，刷新一下浏览器页面，面板应该也会变成`test_node_url`的配置了，你可以验证一下。
